frontend/src/scenes/surveys/surveysLogic.tsx

```diff
@@ -32,6 +34,10 @@ export function getSurveyStatus(survey: Pick<Survey, 'start_date' | 'end_date'>)
     return ProgressStatus.Complete
 }
 
+function hasNextPage(surveys: CountedPaginatedResponse<Survey>): boolean {
+    return surveys.next !== null && surveys.next !== undefined
```

Comment:
is `next` reliable here? Shouldn't we actually check the `count`?

Reply:
both are coming from the server so reliability wise they're the same. I actually used `count` first, but switched to `next` to avoid doing client-side calculations which is clearer imo. 

happy to change if you feel strong about it, no real preference on my side.